pom.xml

```diff
@@ -47,10 +47,31 @@
     </pluginRepositories>
 
     <properties>
+        <axis2.transport.version.1.1.0v15>1.1.0-wso2v15</axis2.transport.version.1.1.0v15>
+        <am.version.2.0.0>2.0.0-SNAPSHOT</am.version.2.0.0>
```

Comment:
shouldn't this be the carbon apim repo version since we do not refer to the product?